app/models/response.rb

```diff
@@ -121,6 +121,7 @@ def self.search_qualifiers(scope)
 
       # this qualifier inserts a placeholder that we replace later
       Search::Qualifier.new(name: "text", col: "responses.id", type: :indexed, default: true),
+      Search::Qualifier.new(name: "shortcode", col: "responses.shortcode", type: :indexed, default: true),
```

Comment:
We talked about this before and I was going that route but then the implementation broke a test in response_search_spec.rb:95

The test broke because there are now 2 columns doing exact match and one of them was not building the correct query for Answers. We are not dealing with answers at all

So I adjusted it so it could fix the query to use OR which was fixed by adding `type: :indexed`